### ISSUE TITLE
Dose this library work with XBee S2C？

### ISSUE THREAD

**Issue by 408881465** [andrewrapp/xbee-arduino#75]
Dose this library work with XBee S2C？

**Comment by JChristensen** [comment]
Yes, I have been using it with S2C modules with no issues.

**Comment by matdeng** [comment]
Does this library work if i use xbee shield that attach with arduino ?? i use xbee s2c

**Comment by dheijl** [comment]
Yes, I used S2C modules with an Arduino Uno (using softwareserial on pins 2 and 3) and with an Adafruit Metro M4 Express (using Serial1 on pins 0 and 1, a hardware UART).

**Comment by ChetanAmrao** [comment]
how do i send zbStatusResponse from a Router using send() to the Coordinator?
I cannot use ZBTxRequest for it, can I? i might be wrong 
I used the Series2_Tx on the Coordinator and it doesn't receive any StatusResponse so i have to resort to using ZB_EXPLICIT_RX_RESPONSE which i want to avoid now

any guidance would really help me thanks
cheers!

**Comment by claudioandres1677** [comment]
CONECTAR EL ARDUINO DUE Y XBEE 2 O  S2C CON ESTÁ LIBRERÍA NO TENDRÉ PROBLEMAS ?

**Comment by claudioandres1677** [comment]
CONNECT THE ARDUINO DUE AND XBEE 2 OR S2C WITH THIS LIBRARY , WILL NOT HAVE PROBLEMS?
